ansible/vagrant/Vagrantfile.Solaris10

```diff
@@ -19,7 +19,7 @@ SCRIPT
 Vagrant.configure("2") do |config|
 
   config.vm.define :adoptopenjdkSol10 do |adoptopenjdkSol10|
-    adoptopenjdkSol10.vm.box = "solaris10"
+    adoptopenjdkSol10.vm.box = "tnarik/solaris10-minimal"
```

Comment:
Is this a linter failure? If not it shouldn't be in this PR

Reply:
ah, this should not be in this PR. i was testing the solaris vagrant must be mixed with branches....